roles/variables/defaults/main.yml

```diff
@@ -433,6 +447,17 @@ kafka_broker_default_listeners: "{
   }{% endif %}{% endif %}
 }"
 
+kafka_controller_listeners: "{
```

Comment:
Do you want to give option to configure customer listeners on controller ?

Reply:
Currently, we don't. 
But there are options/variables given to control listener properties like ssl_enabled, ssl_mutual_auth_enabled, and sasl_mechanism.

Do you foresee a case where someone would want to add more listeners to the controller?